### ISSUE TITLE
Provide feedback to user when profile update fails

### ISSUE THREAD

**Issue by shehbashahab** [WomenWhoCode/WomenWhoCode#430]
One example: https://github.com/WomenWhoCode/WomenWhoCode/blob/master/app/src/main/java/com/womenwhocode/womenwhocode/activities/TimelineActivity.java#L293:L297

We could potentially add the same error message in all areas where the update failed.

**Comment by opticod** [comment]
If we want to provide feedback to users ,rather using log messages we can add snackbar. 
For Release Edition, we will remove all the Log messages and only will keep the snackbar.

**Comment by shehbashahab** [comment]
This needs more thought, but for now let's keep it simple and log the error and give the user the option to retry submission.

In the future, we can consider adding a 'Report bug' option.

**Comment by zassmin** [comment]
yes, and we can tell the user what changes need to be made with a SnackBar.

**Comment by chhavip** [comment]
@zassmin Willing to work on this if required. Please indicate the required message to be displayed in SnackBar
